.github/workflows/rust.yml

```diff
@@ -0,0 +1,32 @@
+name: Rust
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  checks:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Add rustfmt and clippy
+        run: rustup component add rustfmt clippy
+      - name: Install node dependencies
+        run: yarn
+      - name: Expand liquid
+        run: yarn expand-liquid typescript
```

Comment:
Should this be Rust?

Reply:
No, it is JS flavour so it doesn't matter and I can remove it here so we fall back to the default of `vanilla-js`

Reply:
That would look less confusing 😄